src/entities/Proposal/calculateOutcome.ts

```diff
@@ -0,0 +1,95 @@
+import { Env } from '@dcl/ui-env'
+import JobContext from 'decentraland-gatsby/dist/entities/Job/context'
+
+import { SnapshotGraphql } from '../../clients/SnapshotGraphql'
+import { config } from '../../config'
+import { Scores } from '../Votes/utils'
+
+import { INVALID_PROPOSAL_POLL_OPTIONS, ProposalAttributes } from './types'
+
+function sameOptions(options: string[], expected: string[]) {
```

Comment:
What if we use `isEqual` from lodash?

Reply:
Could be, but I'd rather not change this now, I just moved it :)